docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md

```diff
@@ -0,0 +1,714 @@
+
+Weedle Guy:
+- Tail Whip x2 (Tail Whip a third time with 10 ATK)
+- Tackle Spam [Heal if HP < 5]
+
+Swap Squirtle and Nido at some point
+- Toss antidote if you are healing Squirtle before Brock
+- Potion if HP is below 11
+
+Pewter Mart
+- Buy 8 Potions (7 if you bought 5 balls)
+
+Brock:
+- Swap to Squirtle
+- Bubble x2-3 on Geodude [Heal if HP < 8]
+- Switch to Nido, then swap back to Squirtle
+- Bubble x2-3 on Onix [Heal if HP < 7]
+
+## ROUTE 3 SPLIT
+
+Change Options to Set before leaving the Gym
+
+BC1:
+- Leer, HA x2(Tackle)
+- Leer, HA + (HA or Tackle if you can)
+- Leer, HA + Tackle
+
+- Menu: Use a potion if HP < 25 & 
+*Toss antidote if healing before Shorts Guy*
+> with 25-27 hp use HAx2 (+Tackle) on Rat
+
+Shorts Guy:
+- Leer, HA x2  [Heal if HP < 10, @-1 if HP < 16]
+- Leer, HA x2  [Heal if HP < 4, @-1 hp < 5, @-2 if hp < 6]
+> Potion turn 1 of wrap if you don't live a 4 or 5-turn-wrap.
+
+BC2:
+- HA + HA or Tackle
+- HA x3 (+ HA or Tackle)
+- HA x2 (Tackle turn 2 if Weedle hit 2 string shots) (Heal if hp < 4)
+- HA x2 + Tackle or finish with HA if Metapod outspeeds you
+
+- if hp < 4 Menu before the next fight and potionx2
+
+BC3:
+- HA x2
+- HA x3-4(finish with Tackle if you can)
+
++ Catch a Flyer
+  + Tackle then toss a ball at any flyer except L8 Pidgey where you HA then toss a ball
+  > Exception: if only 2 HAs are left, then weaken L8 Pidgey with Leer + Tackle
+
+## MT. MOON SPLIT
+
+**Moon Manip Save**
+- 1- 8
+  + 2 Potions AFTER Paras
+- 9 - 15
+  + 2 Potions
+- 16 - 35
+  + 1 Potion
+- 36+
+  + Don't Heal
+
+Do Route 3 Moon Manip https://pastebin.com/tggXpQRC
+
+>The Lass that you see on screen before entering Mt. Moon during the manip should be stuck facing up; however on Frame 36 & Frame 37 she walks right & faces left, respectively. If you'd like to learn the alternative movement used to manipulate mt. moon on these frames check here (../../../resources/route-3-moon-backups.md)
+
+If you get an igt encounter or fail the manip before the last floor:
+  - [Post Hiker Backup Backup](https://pastebin.com/j5gtY4cy)
+
+**Rocket:**
+
+* 2 HA:
+	- Rat: Leer + HA + Tackle (or PS turn 3 if tail whip hit or PS always KOs)
+	- Zubat: Leer + Tackle + HA (+Tackle)
+* 3 HA:
+	- Rat: HA x2 (+PS)
+	- Zubat: Leer + Tackle + HA (+Tackle)
+* 4 HA:
+	- Rat: HA x2 (+PS)
+	- Zubat: Leer + HA x2
+* 5+ HA:
+	- Rat: HA x2 (+PS)
+	- Zubat: HA x3 (+Tackle)
+
+ + Heal on Zubat if 
+  + @-0 hp < 8 or confused & hp < 17
+  + @-1 hp < 12 or confused & hp <14
+  + @-2 hp < 14; if confused yolo or consider swap on good pace if Bird or Squirtle have low hp
+
+**Moon Menu** immediately after defeating Rocket:  
+  + Toss Pokeball(s) (do not toss if you still need to catch Paras)
+  + Use Rare Candy
+  + Teach Water Gun (TM12) in Slot 2
+  + Use Moon Stone
+  + Teach Mega Punch (TM01) in Slot 1
+
+> Moon Exp: KO a Geodude or Paras if you happen to encounter one while walking up to talk to the Nerd
```

Comment:
Maybe superfluous, mention that you use PS to kill Paras?

Reply:
Yup, makes sense. Added WG geodude as well cause why not